rpc/handlers.go

```diff
@@ -512,7 +523,7 @@ func (h *Handler) MethodsV0_7() ([]jsonrpc.Method, string) { //nolint: funlen
 			Handler: h.SpecVersionV0_7,
 		},
 		{
-			Name:    "juno_subscribeNewHeads",
+			Name:    "starknet_subscribeNewHeads",
```

Comment:
why do you need to update `starknet_subscribeNewHeads` in this PR ?

Reply:
I'll revert 👍 - I based on Han's PR - and then reverted when Anique's PR get merged